src/airtableApi.jsx

```diff
@@ -0,0 +1,77 @@
+import {
+  AIRTABLE_TABLE_NAME,
+  AIRTABLE_BASE_ID,
+  AIRTABLE_API_KEY,
+} from './configuration.jsx';
+
+const tableUrl = `https://api.airtable.com/v0/${AIRTABLE_BASE_ID}/${AIRTABLE_TABLE_NAME}`;
+
+const defaultHeaders = {
+  Authorization: `Bearer ${AIRTABLE_API_KEY}`,
+};
+export const listRecords = async options => {
+  try {
+    let params = {};
+    if (options?.searchText) {
+      params = {
+        ...params,
+        filterByFormula: `SEARCH('${options.searchText.toLowerCase()}', {title})`,
```

Comment:
![image](https://user-images.githubusercontent.com/6400323/224869705-91d885ce-8c64-4655-8535-6a5dde38f776.png)

I found `filterByFormula` in the API documentation.  There are other options like for sorting the records or controlling the number of records returned.